linalg/blockoperator.cpp

```diff
@@ -149,7 +148,7 @@ BlockDiagonalPreconditioner::BlockDiagonalPreconditioner(
    offsets(0),
    ops(nBlocks)
 {
-   ops = static_cast<Operator *>(NULL);
+   ops = static_cast<Operator *>(nullptr);
```

Comment:
```suggestion
   ops = nullptr;
```

Reply:
done in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7